### DISCUSSION TITLE
Working example

### DISCUSSION THREAD

**Discussion by punx120** [JaneWardSandy/eclipse-pde-partial-idea#94] [Q&A] [answered]
Hi,

I'm having issue launching the working example - i got it to compile, but i can't find the application in the run config:
![Run-Debug Configurations_001](https://user-images.githubusercontent.com/8940871/196571432-a05f36d9-2dee-4639-a179-cbb76e4e5c2c.png)

What am i doing wrong?

**Comment by JaneWardSandy** [comment]
In `eclipse`, it can select  one of `product` and `application` to run. And when you only select the `application`, it will auto detect the `product` belonging the `application`.


In this plugins, we need to select both of `product` and `application` manually, if debug the `eclipse` SDK, we can select `org.eclipse.platform.ide` for `product` and `org.eclipse.ui.ide.workbench` for `application`

<img width="717" alt="image" src="https://user-images.githubusercontent.com/15345355/196573894-ec658310-fe21-4781-bcb4-ef7b516a4125.png">

**Reply by punx120** [reply]
I guess my question is how to I start/run the example application?

**Comment by JaneWardSandy** [comment, marked as answer]
select `org.eclipse.platform.ide` for `product` and `org.eclipse.ui.ide.workbench` for `application`, save the run configuration, then click `Run` menu -> `Run / Debug Main` like a normal program. This plugins will auto generate the OSGI development file to launch the `Equinox` with specify `launcher jar`

**Reply by punx120** [reply]
Oh my bad - i thought it was a standalone application - not a eclipse plugin... thank you !
